migrator/migrations/m_172_to_m_173_enable_search_on_api_tokens/oldapitokenpostgresstore/postgres_plugin.go

```diff
@@ -0,0 +1,303 @@
+package postgres
```

Comment:
I was under the impression that these files are autogenerated, or are they not?

Reply:
Not really. The way I did the first postgres -> postgres migration was by copying over the pieces of the store file I needed, and pruning the code I did not need.
In the meantime, the code is supposed to rely on frozen schema data, which in my opinion does make sense. But the code generator support for migration is limited, and does not seem to cover the store part anymore.

Reply:
Ah I understand, was under the initial impression that this was autogenerated.

Do you think it makes sense to add a comment header calling out that the code was copied from the datastore and all unused functions were removed? I know we had that etiquette in the rocksdb migrations, I'd say it makes sense to continue with it here as well?